.ci/validation.sh

```diff
@@ -41,7 +41,7 @@ checkstyle-tester-diff-groovy-patch)
   sed -i'' 's/^guava/#guava/' ../.ci-temp/projects-to-test-on.properties
   sed -i'' 's/#checkstyle/checkstyle/' ../.ci-temp/projects-to-test-on.properties
   groovy diff.groovy -l ../.ci-temp/projects-to-test-on.properties \
-    -c my_check.xml -b master -p patch-branch -r ../.ci-temp/checkstyle
+    -c my_check.xml -b master -p patch-branch -r ../.ci-temp/checkstyle -h true
```

Comment:
@rnveach, i modified the CI to include shallow clone feature. please let me know if understood your concern right.

Reply:
Yes you did.

Does it make more sense if we can configure this by just `-h` alone? It seems redundant to also specify true when this will only be used to make it true.

Reply:
Few lines can use long version of this patamer, just for diversity of testing and example of usage

Reply:
Based on my understanding, i have modified the description of the long version and documentation. Earlier, it required a parameter like true or false. Right now, the idea is to do shallow clone mandatorily, when we have such a parameter in CLI. 

> Few lines can use long version of this patamer, just for diversity of testing and example of usage

done.